packages/sdk/src/controllers/SubscriberController.ts

```diff
@@ -66,21 +66,44 @@ export class SubscriberController {
         callBack && callBack(unit as MeasurementUnit);
     }
 
+
     /**
-     * Listener on the state of the currently selected frame, if this changes, this listener will get triggered with the updates
-     * @param frameLayout Stringified object of FrameLayoutType
+     * @deprecated use `onSelectedFramesLayoutChanged` instead
+     * Listener on the state of the currently selected frames, if this changes, this listener will get triggered with the updates
+     * @param frameLayout Stringified array of FrameLayoutType objects
      */
     onSelectedFrameLayoutChanged = (frameLayout: string) => {
-        const callBack = this.config.onSelectedFrameLayoutChanged;
+        const callBack = this.config.onSelectedFramesLayoutChanged;
+        const frames = JSON.parse(frameLayout);
+        callBack && callBack(frames[0]);
+    };
+
+    /**
+     * Listener on the state of the currently selected frames, if this changes, this listener will get triggered with the updates
+     * @param frameLayout Stringified array of FrameLayoutType objects
+     */
+    onSelectedFramesLayoutChanged = (frameLayout: string) => {
+        const callBack = this.config.onSelectedFramesLayoutChanged;
         callBack && callBack(JSON.parse(frameLayout));
     };
 
     /**
+     * @deprecated use `onSelectedFramesContentChanged` instead
      * Listener on the state of the currently selected frame, it contains some basic information on the type of frame it is
-     * @param frameContent Stringified object of FrameType
+     * @param frameContent Stringified array of Frame objects
      */
     onSelectedFrameContentChanged = (frameContent: string) => {
```

Comment:
Update the comment to reflect that it will be the first frame when multiple are selected